backends/conrod_gfx/src/lib.rs

```diff
@@ -815,14 +815,6 @@ impl From<gfx::PipelineStateError<String>> for RendererCreationError {
     }
 }
 
-impl std::error::Error for RendererCreationError {
-    fn description(&self) -> &str {
-        match *self {
-            RendererCreationError::PipelineState(ref e) => std::error::Error::description(e),
-        }
-    }
-}
```

Comment:
Can we leave the `std::error::Error` implementations and only remove the inner `description` methods instead? Otherwise I think we might break some downstream crates that rely on these error types implementing the `std::error::Error` trait.